src/map/atcommand.c

```diff
@@ -3537,7 +3537,7 @@ ACMD(recallall)
 {
 	struct map_session_data* pl_sd;
 	struct s_mapiterator* iter;
-	int count;
+	int unauthorised_count, idle_count;
```

Comment:
better move this variabled definition into lines where they assigned first time. to line 3549